nuget.config

```
@@ -2,5 +2,6 @@
 <configuration>
     <packageSources>
         <add key="Project Packages" value="src/Files.App/Assets/Libraries/" />
+        <add key="Lab-Windows" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json" />
     </packageSources>
 </configuration>
```

```suggestion
<configuration>
    <packageSources>
        <clear />
        <add key="NuGet Gallery" value="https://api.nuget.org/v3/index.json" />
        <add key="Project Packages" value="src/Files.App/Assets/Libraries/" />
        <add key="Toolkit Labs" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json" />
    </packageSources>
</configuration>
```

This could potentially fix issues on some computers where the user has a custom NuGet setup.

Also rename to `Toolkit Labs` to be consistent with what Microsoft calls it